packages/libs/api/sdk/codegen.yml

```diff
@@ -6,4 +6,7 @@ generates:
     plugins:
       - typescript
       - typescript-operations
-      - typed-document-node
+      - typescript-document-nodes
+  src/graphql/fragmentMatcher.ts:
```

Comment:
I think in the near future we want to standardize the return types. I added this codegen plugin so we can make sure we keep consistency between multiple requests